plugins/flowdock/test/models/stage_test.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../test_helper'
+
+describe Stage do
+
+  subject { stages(:test_staging) }
+
+  before do
+    subject.notify_email_address = "test@test.ttt"
+    subject.flowdock_flows = [FlowdockFlow.new(name: "test", token: "abcxyz", stage_id: subject.id, enabled: false)]
+    subject.save
+  end
+
+  describe 'flowdock notifications' do
```

Comment:
I usually do `describe "#send_flowdock_notifications?"` so it's easy to see which block tests what